src/python_testing/TC_OVENOPSTATE_2_3.py

```diff
@@ -0,0 +1,159 @@
+#
+#    Copyright (c) 2023 Project CHIP Authors
+#    All rights reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+import logging
```

Comment:
please add both tests to test.yaml so they run in the CI

Reply:
connectedhomeip/src/app/tests/suites/certification/tests.yaml . To this file?

Reply:
@prakashece https://github.com/project-chip/connectedhomeip/blob/039cbab7aeb6387d56b6196249b45ea71350752a/.github/workflows/tests.yaml#L452